app/controllers/registration_controller.rb

```diff
@@ -33,12 +33,11 @@ def workflow_list
   # Data route to return all the registration collections listed for the given APO
   # @option params [String] `:apo_id` the druid for the APO
   # @option params [String] `:truncate` word boundary truncation limit
+  # @return [Hash] keys represent collection druids, values represent collection titles, alpha-sorted except for the leading "None" option
```

Comment:
I think I understand what you mean, but what about saying what the keys actually are?